pdfalyzer/util/argument_parser.py

```diff
@@ -50,6 +50,10 @@
                     const='bin',
                     help='extract all binary streams in the PDF to separate files (requires pdf-parser.py)')
 
+export.add_argument('-json',
```

Comment:
let's add the long form version of the argument so it's consistent with yaralyzer, so it would be

```python
export.add_argument('-json', '--export-json',
                    action='store_const',
                    const='json',
                    help='export analysis results to JSON files preserving PDF structure')
```

(and btw if you're feeling particularly ambitious this feature would be a great addition to [yaralyzer](https://github.com/michelcrypt4d4mus/yaralyzer)... if the change was made there the option would propagate to pdfalyzer)